portal-ecs.yml

```diff
@@ -164,12 +167,25 @@ Parameters:
     Description: OpenGraph title property used for home page
   OpenGraphDescription:
     Type: String
-    Description: OpenGraph descriptio property used for home page and other pages without
-      their own Description
+    Description: OpenGraph description property used for home page and other
+      pages without their own Description
+
   OpenGraphImageURL:
     Type: String
-    Description: OpenGraph image property used for home page ad other pages without their
-      own description
+    Description: OpenGraph image property used for home page ad other pages
+      without their own description
+
+  SubnetIds:
+    Type: String
+    Description: Subnet must be a publicly accessible subnet on the main VPC
+    # QA Value: 'subnet-13a1133d,subnet-2d925713'
+    Default: 'subnet-7fb16326'
```

Comment:
I looked at how production is setup. The production cluster has subnets of `subnet-7fb16326,subnet-09c2dd40`, so that includes the `subnet-7fb16326` which was explicitly used in the elastic-search stack. 
Based on that I'm pretty sure we can use the same subnets the cluster is using for the elastic-search instance. 

So I'm pretty sure we can drop this explicit parameter, and just pass `{ 'Fn::ImportValue': !Sub '${ClusterStackName}-ClusterSubnets' }` to the elastic-search stack.